src/content/blog/ddev-name-resolution-wildcards.md

```diff
@@ -0,0 +1,101 @@
+
+## Can I set up a domain name on my local network or company network?
+
+Yes, this is exactly the same. Normally you would have every DDEV-related domain name return `127.0.0.1` for the DNS name you're supporting. However, some people use this technique to [share DDEV sites inside a local network](https://ddev.readthedocs.io/en/stable/users/topics/sharing/#exposing-a-host-port-and-providing-a-direct-url). 
+
+## Can I set up my own domain in DNS like `ddev.site`?
+
+Yes, if you have the ability to configure an internet DNS zone, you can set it up the same way `ddev.site` is set up, resolving to `127.0.0.1` and with a wildcard that resovles to `127.0.0.1`.
+
+## What happens on WSL2 and why do I have to take action on the Windows side?
+
+On WSL2, when you don't have internet, or your DNS is broken, or you're using a hostname that can't be looked up in DNS, DDEV has to punt the hosts file change over to the Windows side of things, because your *browser* is almost always running on Windows, so editing the hosts file on the Windows side is what has to happen. The way DDEV currently does this is requests a **Windows** version of `ddev.exe` to escalate and edit the hosts file. That's why WSL2 installations also ask you to install Windows DDEV using `choco install -y ddev`.
+
+## No matter what, DDEV always wants to edit my hosts file (No DNS Rebinding)
+
+There is one DNS server configuration that disallows looking up any hostname that resolves to `127.0.0.1`. It's called preventing [DNS Rebinding](https://en.wikipedia.org/wiki/DNS_rebinding), and it happens most often to European users who have Fritzbox routers. The fix for this is covered in the [DDEV Docs](https://ddev.readthedocs.io/en/stable/users/usage/troubleshooting/#dns-rebinding-prohibited-mostly-on-fritzbox-routers) and it's easy and will make your life much easier.
+
+However, if you don't have control of your router, there's another easy fix, and that is to use a trusted DNS server outside your network. To do this, you would change your DNS server on your computer to `1.1.1.1` for Cloudflare's public DNS server, or `8.8.8.8` for Google's public DNS. Those will both do ordinary DNS resolution. There are lots of articles on the internet explaining how to change your computer's DNS servers, [here's one of them](https://www.hellotech.com/guide/for/how-to-change-dns-server-windows-mac). Unless you're on a corporate network with private DNS, this is unlikely to cause you any trouble and may resolve other difficulties, like removing some advertisements.
+
+## How can I tell if `ddev.site` name resolution is broken?
+
+Try `ping -c 1 xxx.ddev.site`. It it pings `127.0.0.1` and gets a response, then either  `xxx.ddev.site` name resolution is working or you have added it to your hosts file.
```

Comment:
```suggestion
Try `ping -c 1 t.e.s.t.ddev.site`. If it doesn't ping `127.0.0.1`, it is broken. Please see the last question for help troubleshooting.
```
Using multiple subdomains that are not likely to be entered in hosts will better test the resolution.